### PR TITLE
Update base software for windows 21.08 release

### PR DESCRIPTION
**Pull request by deadeyegoodwin**

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -382,13 +382,11 @@ def pytorch_cmake_args(images):
 
 def onnxruntime_cmake_args(images, library_paths):
     cargs = [
+        '-DTRITON_ENABLE_ONNXRUNTIME_TENSORRT=ON',
```
Comment (CoderHam) [inline build.py:385]: We should hold off on enabling TRT support for jetpack till I add in the updated ORT build (1.8.1) which enables TRT 8 support.

File: docs/build.md [hunk 2]
```diff
@@ -229,7 +229,7 @@ cuDNN and TensorRT versions and place them in the local directory.
 
 * For cuDNN the CUDNN_VERSION and CUDNN_ZIP arguments indicate the
```
Comment (CoderHam) [inline docs/build.md:230]: Add a comma after "For cuDNN"

File: Dockerfile.win10.min [hunk 3]
```diff
@@ -86,6 +65,27 @@ RUN vcpkg.exe update
...
+                   cublas_${CUDA_MAJOR}.${CUDA_MINOR} cublas_dev_${CUDA_MAJOR}.${CUDA_MINOR} \
+                   cufft_${CUDA_MAJOR}.${CUDA_MINOR} cufft_dev_${CUDA_MAJOR}.${CUDA_MINOR} \
+                   curand_${CUDA_MAJOR}.${CUDA_MINOR} curand_dev_${CUDA_MAJOR}.${CUDA_MINOR} \
+                   cusolver_${CUDA_MAJOR}.${CUDA_MINOR} cusolver_dev_${CUDA_MAJOR}.${CUDA_MINOR} \
+                   cusparse_${CUDA_MAJOR}.${CUDA_MINOR} cusparse_dev_${CUDA_MAJOR}.${CUDA_MINOR} \
+                   visual_studio_integration_${CUDA_MAJOR}.${CUDA_MINOR}"
+ARG CUDA_INSTALL_ROOT_WP="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v${CUDA_MAJOR}.${CUDA_MINOR}"
+
+ARG TENSORRT_VERSION=8.0.1.6
+ARG TENSORRT_ZIP=TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-11.3.cudnn8.2.zip
```
Comment (CoderHam) [inline Dockerfile.win10.min:84]: Should this not be 11.4 instead of 11.3?

**Review by CoderHam — CHANGES_REQUESTED** [review]

**Comment by deadeyegoodwin** [comment]
This change is incorporated into r21.08 branch. Closing.

**Pull request closed without merging**